docs/se/reactivemessaging/04_kafka.adoc

```diff
@@ -0,0 +1,165 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2020 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Kafka Connector
+:toc:
+:toc-placement: preamble
+:description: Reactive Messaging support for Kafka in Helidon SE
+:keywords: helidon, se, messaging, kafka
+
+== Reactive Kafka Connector
+Connecting streams to Kafka via Reactive Messaging just can't be easier. 
+
```

Comment:
Connecting streams to Kafka with Reactive Messaging couldn't be easier.

Reply:
thx fixed